chef/cookbooks/ceph/providers/pool.rb

```diff
@@ -24,8 +24,8 @@ def get_pg_num(ceph_conf, admin_keyring)
 end
 
 def create_pool(ceph_conf, admin_keyring, pool_name, pg_num)
-  Chef::Log.info("Creating ceph pool '#{pool_name}' with pg number '#{pg_num}'")
-  cmd = "ceph -k #{admin_keyring} -c #{ceph_conf} osd pool create '#{pool_name}' '#{pg_num}'"
+  Chef::Log.info("Creating ceph pool \"#{pool_name}\" with pg number #{pg_num}")
+  cmd = "ceph -k #{admin_keyring} -c #{ceph_conf} osd pool create \"#{pool_name}\" #{pg_num}"
```

Comment:
That's actually worse ;-) You still need to escape quotes (`"` now), but on top of that, you need  to escape `$` :-)